### PR TITLE
chore: Add card network assets for Cartes Bancaires

### PR DESCRIPTION
**Pull request by jnewc**
CHKT-1755

# What this PR does

Adds a logo, icon, and card icon asset for Cartes Bancaires

# Notable decisions & other stuff

* Adds a new method for fetching card network assets
* Reorganises card icons into their own asset folder

# Before merging

_QA_

- [x] I manually tested this with a demo application (simulator)
- [x] I wrote unit tests for each new util-like function

### CODE REVIEW

**Review by semirp — APPROVED** [review]

**Review by NQuinn27 — APPROVED** [review]

File: Debug App/Tests/Unit Tests/Assets/AssetsManagerTests.swift [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        SDKSessionHelper.tearDown()
+    }
+
+    func testCardAssets() throws {
+        XCTAssertNotNil(try AssetsManager.getCardNetworkAssets(for: .cartesBancaires))
+        XCTAssertNotNil(try AssetsManager.getCardNetworkAssets(for: .discover))
+        XCTAssertNotNil(try AssetsManager.getCardNetworkAssets(for: .masterCard))
+        XCTAssertNotNil(try AssetsManager.getCardNetworkAssets(for: .visa))
+        XCTAssertNotNil(try AssetsManager.getCardNetworkAssets(for: .amex))
+
```
Comment (borisprimer) [inline Debug App/Tests/Unit Tests/Assets/AssetsManagerTests.swift:30]: extra new line?

**Review by borisprimer — APPROVED** [review]
LGTM

**Review by borisprimer — APPROVED** [review]

File: Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Managers/AssetsManager.swift [hunk 2]
```diff
@@ -23,6 +24,25 @@ extension PrimerHeadlessUniversalCheckout {
             return UIImage(named: "\(cardNetwork.rawValue)-logo-colored", in: Bundle.primerResources, compatibleWith: nil)
         }
         
+        public static func getCardNetworkAssets(for cardNetwork: CardNetwork) throws -> PrimerCardNetworkAsset? {
```
Comment (NQuinn27) [inline Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Managers/AssetsManager.swift:27]: Nit-pick: This method is named `getCardNetworkAssets` but returns a singular `PrimerNetworkAsset`, potentially misleading 🤔

File: Debug App/Tests/Unit Tests/Assets/AssetsManagerTests.swift (hunk 1 above)
Comment (jnewc) [reply Debug App/Tests/Unit Tests/Assets/AssetsManagerTests.swift:30]: I do this to separate the `Nil`s from the `NotNil`s. The `Nil` case here is just to prove the 'null hypothesis' fails, as we mostly care that the assets are available for the right networks

**Pull request merged**